src/main/riddl/Gateway/MemberGateway/memberStories.riddl

```diff
@@ -24,11 +24,11 @@ story RegisterMember is {
         briefly "Member clicks on the enroll button",
       step from actor App send command Member.EnrollMember to actor GatewayAPI //ImprovingApp.Gateway.MemberGateway.Member
         briefly "App tells the Gateway to enroll the member",
-    //   step from actor GatewayAPI "Register Member" to entity ImprovingApp.MemberContext.Member
+    //   step from actor GatewayAPI "Register Member" to entity ImprovingApp.MembersContext.Member
     //     briefly "add new organization",
-    //   step from entity ImprovingApp.MemberContext.Member "MemberRegistered" to pipe ImprovingApp.MemberContext.MemberEvents
+    //   step from entity ImprovingApp.MembersContext.Member "MemberRegistered" to pipe ImprovingApp.MembersContext.MemberEvents
     //     briefly "organization added"
-    //   step from pipe ImprovingApp.MemberContext.MemberEvents "MemberEnrolled" to actor App
+    //   step from pipe ImprovingApp.MembersContext.MemberEvents "MemberEnrolled" to actor App
     }
```

Comment:
Wouldn't it be better also to uncomment these so riddlc can validate the pathIds and the documentation can represent the relationships they imply?